modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_vbms_uploader.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'pdf_generator_service/pdf_client'
+require 'bd/bd'
+
+module ClaimsApi
+  module V2
+    class DisabilityCompensationVBMSUploader < DisabilityCompensationClaimServiceBase
```

Comment:
I have thoughts on this. I know people refer to this as "VBMS Upload" because it ultimately ends up in VBMS, but this is confusing to me. From our standpoint, this does not submit in any way shape or form to VBMS, it submits to Benefits Documents. That does, ultimately, hit VBMS but we are submitting to Benefits Documents. 

I have a strong desire to rename this to better align, but that would take reworking services as well to make it meaningful. 🤔 Thoughts? Ideas? Feedback? I just feel like people may see this and go "oh, they're uploading to VBMS" which we are not directly doing (indirectly we are, so that's why I'm open for discussing).

Reply:
@FonzMP 100% on board with getting these namings right at the beginning.  So what we would be looking for is `DisabilitycompensationBenefitDocumentsUpload` , feel right?